intermediate-week-1/app.js

```diff
@@ -4,6 +4,7 @@ const cors = require('cors');
 const errorHandler = require('./error/errorHandler');
 const AppError = require('./error/appError');
 const morgan = require('morgan');
+const { provideErrorHandler } = require('./middlewares/errorHandler');
```

Comment:
I''m not too sure what this does. But it seems its supposed to handle any errors that come up. 

But putting it so high up means that it is the first middleware the your requrests will meet. 

Error handling middlewrae should be the last ones you add .